README.md

```diff
@@ -2,19 +2,39 @@
 
 ## What is DColor?
 
-DColor is a Python3 module for visualizing complex-valued functions using a [Domain Coloring](https://en.wikipedia.org/wiki/Domain_coloring) scheme.
+DColor is a Python 3 module for visualizing complex-valued functions using a [Domain Coloring](https://en.wikipedia.org/wiki/Domain_coloring) scheme.
 
 ## Requirements
 
-DColor leverages two well-known Python libraries: Numpy and Matplotlib. Before being able to use DColor, you must install these using the following commands:
+DColor leverages two well-known Python libraries: [NumPy](https://numpy.org/) and [Matplotlib](https://matplotlib.org/).
 
-`$ python3 -m pip install -U numpy`
+### Old (`pip`)
 
-`$ python3 -m pip install -U matplotlib`
+Before being able to use DColor, you must install these using the following commands:
 
-## clone the repository
+```bash
+python3 -m pip install -U numpy
+python3 -m pip install -U matplotlib
+git clone https://github.com/hernanat/dcolor.git
+```
 
-`git clone https://github.com/hernanat/dcolor.git`
+### New (`poetry`)
```

Comment:
ty

Reply:
No problem, I'm actually new to Poetry too, as of this year.

One thing you may have noticed further down is the markdown comment about generating `dist/`, which, along with virtualenvs, is one of the biggest things Poetry helps with. I'm admittedly rather weak when it comes to Python distribution, but it doesn't seem common to include wheelfiles in the Git project structure (at least, there aren't any in [matplotlib](https://github.com/matplotlib/matplotlib)).